CHANGELOG.md

```diff
@@ -1,12 +1,27 @@
 ## Changelog
 
+### 2021.07.1
+
+**Released**: 2021.07.21
+
+**Commit Delta**: [Change from 2021.06.1 release](https://github.com/plus3it/spel/compare/2021.06.1...2021.07.1)
+
+**Manifests**: <https://github.com/plus3it/spel/blob/2021.07.1/manifests>
+
+**Summary**:
+
+*   aws-cli v1 is now installed using python3
+*   "Extra" packages updated in this release:
+    - aws-cli/1.20.4
```

Comment:
CentOS is showing `1.20.3` for aws-cli v1.

Reply:
Well that's weird, must have caught them in between releases. The RHEL build failed the first time around, so there's a couple hours in between...

Reply:
Updated!